lib/mapper.js

```diff
@@ -164,14 +168,16 @@ function createBaseTrackProperties(track, settings) {
   if (settings.prefixProperties) {
     properties = prefix(track.event(), properties);
   }
-
-  return extend(
-    properties,
-    {
+  // Only apply userAgent property on `visited site` event
+  if (track.event() === 'visited site') {
+    properties._userAgent = userAgent || track.proxy('context.userAgent');
+  }
```

Comment:
@paulgoleary Per the line code changes above, I want to confirm we (on the kissmetrics end) only parse `_userAgent` for the event `visited site`, and thus we should only send the `_userAgent` string in the segment kissmetrics-integration when the event being tracked is `visted site`.  This update is per the following user story: https://www.pivotaltracker.com/n/projects/1392800/stories/109322764

cc: @percyhanna